pendant/patterns/404.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Pattern Name: A 404 page
```

Comment:
We don't register this so an empty pattern shows up in the site editor. I think it's fine to do this in a follow-up, unless you wanted to do it now...

Reply:
We're still waiting for [this](https://github.com/WordPress/gutenberg/pull/36751) to land at which point this should start working.  It doesn't work now but I hope it will before we start in on patterns (and the 404 page).   This was copied from @MaggieCabrera 's new boilerplate theme.

My 'druthers is to bring it in now.

Reply:
The pattern PR in GB is going to be merged soon, and it most likely will make it to dotcom before the theme is ready to launch, but we can of course make a patch in the meantime.